spec/component_guide/component_index_spec.rb

```diff
@@ -103,6 +103,7 @@
 //= require govuk_publishing_components/components/layout-header
 //= require govuk_publishing_components/components/layout-super-navigation-header
 //= require govuk_publishing_components/components/print-link
+//= require govuk_publishing_components/components/skip-link
```

Comment:
While I'm looking at this - is there anything we can chance to make these  (and polyfills) _not_ look like comments?